packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,514 @@
+<MenuTrigger>
+  <ActionButton>
+      Edit
+  </ActionButton>
+  <Menu>
+    <Item>Cut</Item>
+    <Item>Copy</Item>
+    <Item>Paste</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+If the Menu is open within a MenuTrigger it will close on blur or scroll events.
+
+### Selection
+
+The Menu's selected Item key is propagated as the selection via the event [`onAction`](#onaction).
+
+The `defaultSelectedKeys` prop can be used to preselect Menu Items in the Menu
+placing selection of a Menu in an uncontrolled state. Alternatively, the
+`selectedKeys` prop preselects an Item in the Menu placing Item selection in a
+controlled state.
+
+```tsx example
+<div>Edit (Controlled)</div>
+<Menu
+  selectionMode="single"
+  selectedKeys={['copy']}
+  width="200px">
+  <Item uniqueKey="cut">Cut</Item>
+  <Item uniqueKey="copy">Copy</Item>
+  <Item uniqueKey="paste">Paste</Item>
+</Menu>
+<div>Edit (Uncontrolled)</div>
+<Menu
+  selectionMode="single"
+  defaultSelectedKeys={['paste']}
+  width="200px">
+  <Item uniqueKey="cut">Cut</Item>
+  <Item uniqueKey="copy">Copy</Item>
+  <Item uniqueKey="paste">Paste</Item>
+</Menu>
+```
+
+The `selectionMode` prop specifies how many Menu Items can be selected, with
+the options of a single Menu Item, multiple Menu Items, or disabling selection
+entirely (default).
+
+```tsx example
+<div>Show (multiple)</div>
+<Menu
+  selectionMode="multiple"
+  defaultSelectedKeys={['Sidebar', 'Console']}
+  width="200px">
+  <Item uniqueKey='Sidebar'>Sidebar</Item>
+  <Item uniqueKey='Searchbar'>Searchbar</Item>
+  <Item uniqueKey='Tools'>Tools</Item>
+  <Item uniqueKey='Console'>Console</Item>
+</Menu>
+<div>Selection Mode None</div>
+<Menu
+  selectionMode="none"
+  width="200px">
+  <Item uniqueKey="cut">Cut</Item>
+  <Item uniqueKey="copy">Copy</Item>
+  <Item uniqueKey="paste">Paste</Item>
+</Menu>
+```
+
+### Sections
+
+Menus may have Sections which can be used to wrap groups of Items. Each
+`Section` takes a `title` and `uniqueKey` prop.
+
+#### Static Items
+
+```tsx example
+<div>Edit</div>
+<Menu width="200px">
+    <Section uniqueKey="rollback" title="Rollback Options">
+      <Item uniqueKey="undo">Undo</Item>
+      <Item uniqueKey="redo">Redo</Item>
+    </Section>
+    <Section uniqueKey="select" title="Selected Text Options">
+      <Item uniqueKey="cut">Cut</Item>
+      <Item uniqueKey="copy">Copy</Item>
+      <Item uniqueKey="paste">Paste</Item>
+    </Section>
+</Menu>
+```
+
+#### Dynamic Items
+
+Sections should be populated with dynamic Items from a hierarchical data
+structure. Section takes an array of data using the `items` prop.
+
+```tsx example
+<div>File Types</div>
+<Menu
+  items={[{name: 'Docs', children: [{name: 'PDF'}]}, {name: 'Images', children: [{name: 'jpeg'}, {name: 'png'}, {name: 'tiff'}]}]}
+  itemKey="name"
+  width="200px">
+  {item => (
+    <Section items={item.children} title={item.name}>
+      {item => <Item>{item.name}</Item>}
+    </Section>
+  )}
+</Menu>
+```
+
+### Sematic Elements
```

Comment:
Do we want to avoid this term given yesterday's conversation?

Reply:
I was copying the `menu with semantic elements` story from Menu's storybook.